zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/ZkTestBase.java

```diff
@@ -143,7 +143,11 @@ protected ZkServer startZkServer(final String zkAddress) {
     int port = Integer.parseInt(zkAddress.substring(zkAddress.lastIndexOf(':') + 1));
     ZkServer zkServer = new ZkServer(dataDir, logDir, defaultNameSpace, port);
     System.out.println("Starting ZK server at " + zkAddress);
-    zkServer.start();
+    try {
+      zkServer.start();
+    } catch (Exception e) {
```

Comment:
What's the reason to catch exception and print? I guess you want to have the stacktrace for debugging. But please be noted that if exception is thrown, this change will return a zkServer that's not started. I don't think this is what you expected, is it?
I guest keeping it to throw exception so the test would not proceed.

Reply:
You are right. Let me re-throw the exception to prevent the test from proceeding.

Without printing it out here, the problem is that Intellij will not print out the exception here. Thus, it seems to be a mystery as why all the test is skipped.

Reply:
Revert this part.